org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/settings/SettingsTest.java

```diff
@@ -168,7 +168,7 @@ public void formatSettings() {
 		formattingOptions.setTabSize(5);
 		formattingOptions.setInsertSpaces(false);
 
-		XMLFormattingOptions xmlFormattingOptions = new XMLFormattingOptions(formattingOptions);
+		XMLFormattingOptions xmlFormattingOptions = new XMLFormattingOptions(formattingOptions, false);

```

Comment:
I did this because since `initializeDefaultSettings` calls `super.setTabSize(DEFAULT_TAB_SIZE);`, when merging `formattingOptions`, the `tabSize` will not get updated because it has already been set: https://github.com/eclipse/lemminx/blob/55e92f3c0f51717787f2c3b161181dc2654c293d/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/settings/XMLFormattingOptions.java#L346-L356